src/greentea/gtea/greentea_hooks.py

```diff
@@ -2,43 +2,38 @@
 # Copyright (c) 2021 Arm Limited and Contributors. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
-
+"""Available hooks for greentea."""
 import os
 import re
 import json
+
 from subprocess import Popen, PIPE
 from .greentea_log import gt_logger
 
-"""
-List of available hooks:
-"""
 
-
-class GreenteaTestHook(object):
-    """! Class used to define"""
+class GreenteaCliTestHook:
+    """Class used to define a hook which will call command line program."""
 
     name = None
-
-    def __init__(self, name):
-        self.name = name
-
-    def run(self, format=None):
-        pass
-
-
-class GreenteaCliTestHook(GreenteaTestHook):
-    """! Class used to define a hook which will call command line program"""
-
     cmd = None
 
     def __init__(self, name, cmd):
-        GreenteaTestHook.__init__(self, name)
+        """Initialise CLI Test Hook.
+
+        Args:
+            name: Name of the hook.
+            cmd: Command to be ran.
```

Comment:
Shouldn't it be "command to be run"? Has the command already been executed at this point?